dspace/config/crosswalks/oai/transformers/openaire4.xsl

```diff
@@ -0,0 +1,372 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Following OpenAIRE Guidelines 4 -->
+<xsl:stylesheet version="1.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+    xmlns:doc="http://www.lyncode.com/xoai">
+    <xsl:output indent="yes" method="xml" omit-xml-declaration="yes"/>
+
+    <xsl:template match="@*|node()">
+        <xsl:copy>
+            <xsl:apply-templates select="@*|node()"/>
+        </xsl:copy>
+    </xsl:template>
+ 
+    <!-- Formatting dc.date.issued -->
+    <xsl:template
+        match="/doc:metadata/doc:element[@name='dc']/doc:element[@name='date']/doc:element[@name='issued']/doc:element/doc:field/text()">
+        <xsl:call-template name="formatdate">
+            <xsl:with-param name="datestr" select="."/>
+        </xsl:call-template>
+    </xsl:template>
+
+    <!-- Modifying and normalizing dc.language -->
```

Comment:
It might be worth noting in this comment that specifically you are normalizing it into ISO 639-3 (from ISO 639-1).